dogfood/tester/disruptions.go

```diff
@@ -0,0 +1,201 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package main
+
+import (
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Globals
+
+var SELECTOR = []string{"app", "chaos-dogfood-client"}
+var CONTAINER = "client-deploy"
+
+// Network Disruptions
+var network1 = v1beta1.Disruption{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "e2etest-network1",
+		Namespace: "chaos-engineering",
+	},
+	Spec: v1beta1.DisruptionSpec{
+		Count: &intstr.IntOrString{Type: intstr.Int, IntVal: 1},
+		Unsafemode: &v1beta1.UnsafemodeSpec{
+			DisableAll: true,
+		},
+		Selector:   map[string]string{SELECTOR[0]: SELECTOR[1]},
+		Containers: []string{CONTAINER},
+		Duration:   "3m",
+		Network: &v1beta1.NetworkDisruptionSpec{
+			Hosts: []v1beta1.NetworkDisruptionHostSpec{
+				{
+					Host:     "chaos-dogfood-server.chaos-demo.svc.cluster.local",
+					Port:     50051,
+					Protocol: "tcp",
+				},
+			},
+			Drop:           30,
+			Corrupt:        0,
+			Delay:          0,
+			BandwidthLimit: 0,
+		},
+	},
+}
+
+var network2 = v1beta1.Disruption{
+	ObjectMeta: metav1.ObjectMeta{
+		Name:      "e2etest-network2",
+		Namespace: "chaos-engineering",
+	},
+	Spec: v1beta1.DisruptionSpec{
+		Count: &intstr.IntOrString{Type: intstr.Int, IntVal: 1},
+		Unsafemode: &v1beta1.UnsafemodeSpec{
+			DisableAll: true,
+		},
+		Selector:   map[string]string{SELECTOR[0]: SELECTOR[1]},
+		Containers: []string{CONTAINER},
+		Duration:   "3m",
+		Network: &v1beta1.NetworkDisruptionSpec{
+			Hosts: []v1beta1.NetworkDisruptionHostSpec{
+				{
+					Host:     "chaos-dogfood-server.chaos-demo.svc.cluster.local",
```

Comment:
this is re-used, could we extract it to a const?

Reply:
I think we could also extract more of the disruption spec parameters into their own variables / const, as it seems we're re-using most of it (object meta namespace, spec counts, unsafemode, duration, whole hosts). What do you think?